frontend/components/app.jsx

```diff
@@ -53,13 +53,13 @@ class App extends React.Component {
     return (
       <div>
         <div id="puzzle-container">
-          <Route path="/home" component={() => <Home {...this.state} />}></Route>
-          <Route path="/main" component={() => <Main {...this.state} />} />
-          <Route path="/puzzle1" component={() => <Puzzle {...this.state} component={Puzzle1} />} />
-          <Route path="/puzzle2" component={() => <Puzzle {...this.state} component={Puzzle2} />} />
-          <Route path="/puzzle3" component={() => <Puzzle {...this.state} component={Puzzle3} />} />
-          <Route path="/puzzle4" component={() => <Puzzle {...this.state} component={Puzzle4} />} />
-          <Route path="/puzzle5" component={() => <Puzzle {...this.state} component={Puzzle5} />} />
+          <Route path="/home" render={() => <Home {...this.state} />}></Route>
```

Comment:
does this functionality do the same?

Reply:
yea, the difference is that we are telling Route that this is a render function so that they know they just need to call this function rather than create a component